cv32e40s/tests/programs/custom/umode/umode.c

```diff
@@ -0,0 +1,73 @@
+// TODO:ropeders license header
+
+
+#include <stdio.h>
+#include <stdlib.h>
+
+
+extern volatile int  myfunc(void);
+//extern volatile void run_in_umode(int (*funptr)(void));
+extern volatile int  run_in_umode(int (*funptr)(void));
+extern volatile void setup_pmp(void);
+extern volatile void set_mmode(void);
+
+extern volatile uint32_t gbl_request_mmode = 0;
+
+
+__attribute__ ((interrupt ("machine")))
```

Comment:
`umode/` is some example code that I wrote before. Do we want to include it here?

Do you know what state it is in and if it works well?
If this code is good, we might want to (add a license header and) keep it as a simple example of using user mode.
But it is not otherwise so useful for testing purposes now that we have these new tests.

Reply:
No I believe this was just for reference, nice catch. Can we delete this here or should I push the change from local?

Reply:
Whether github supports it in-browser, I don't know. Pushing the deletion as a commit is probably good.